src/pages/content-edit/invalid-msg.js

```diff
@@ -0,0 +1,80 @@
+import m from "mithril";
+
+import css from "./invalid-msg.css";
+
+// A mosly-dumb controller is required here so we can retain some state 
```

Comment:
This comment makes me feel like it's probably not worth the effort to have it transition then.